css/amp.css

```diff
@@ -221,6 +221,10 @@ html.i-amphtml-ios-embed.i-amphtml-ios-overscroll > #i-amphtml-wrapper {
   bottom: auto !important;
 }
 
+.i-amphtml-layout-fluid {
```

Comment:
nit: can we move this to where all other layout type css are defined.  somewhere here maybe.

Reply:
sure where to? I thought it was right below most of the rest

Reply:
Right below `i-amphtml-layout-flex-item` https://github.com/ampproject/amphtml/blob/master/css/amp.css#L206?

Reply:
done